### PR TITLE
1.12, 1.8.8, and 1.8

### PR DESCRIPTION
**Pull request by IsaiahPatton**
Mappings for 1.12, 1.11.2, 1.8.8, and 1.8

- mappings/1.12.tiny
- mappings/1.11.2.tiny
- mappings/1.8.8.tiny
- mappings/1.8.tiny

### CODE REVIEW

**Comment by shedaniel** [comment]
wait a bit @hYdos don't merge this, I need to fix the build script before you get 6 hours of GitHub actions

**Comment by AMereBagatelle** [comment]
I believe we already have matches down through 1.12, we only require the `.tiny` files to be generated.

**Comment by IsaiahPatton** [comment]
> I don't particularly care about the 1.8 matches, but we already do have matches for 1.12.2-1.12 snapshot by snapshot. If you would like to generate the tiny files for those, that would be very appreciated, but I request that you work based on that rather than submit a match that would only replace it. (Sorry if this didn't quite work how I expected, I am pretty bad at github PRs)

I can only find snapshot matches not the 1.12 release matches. Except 1.11.2 -> 1.12, but the only 1.11 is 1.12 -> 1.11 so there was no way to get release 1.12 matches.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .gradle/
 build/
-.idea/
\ No newline at end of file
+.idea/
+counter.txt
```
Comment (shedaniel) [inline .gitignore:4]: we need the counter

**Pull request merged**